cmd/repl/repl.go

```diff
@@ -64,6 +68,57 @@ func header(head map[string]interface{}) ([]string, error) {
 	return cols, nil
 }
 
+func imageCat(s string, isTerminal bool) bool {
+	if !(isTerminal && strings.HasPrefix(s, "<div")) {
+		return false
+	}
+	match := regexp.MustCompile(`base64,(.*)'`).FindStringSubmatch(s)
+	if len(match) == 2 {
+		image, e := base64.StdEncoding.DecodeString(match[1])
```

Comment:
May I know what format image is? Is it PNG or sixel? Who is supposed to encode the image? SQLFlow server?

Reply:
It's a `png` to be shown on a  jupyter notebook.